pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/service/Google2FAGroupChecker.scala

```diff
@@ -10,7 +11,7 @@ import com.google.api.services.admin.directory.{Directory, DirectoryScopes}
 import scala.collection.JavaConverters._
 import com.gu.pandomainauth.model.{AuthenticatedUser, Google2FAGroupSettings}
 
-class GroupChecker(config: Google2FAGroupSettings, bucket: S3Bucket) {
```

Comment:
I deleted the S3Bucket class since it provided no value other than very minimal wrappers around the S3 client